projects/core/directives/dropdown/dropdown-context.directive.ts

```diff
@@ -44,6 +62,48 @@ export class TuiDropdownContextDirective extends TuiRectAccessor {
         this.driver.next(false);
     }
 
+    @HostListener('touchmove.silent', [
+        '$event.touches[0].clientX',
+        '$event.touches[0].clientY',
+    ])
+    onTouchMove(x: number, y: number): void {
+        if (!this.isIOS) {
+            return;
+        }
+
+        this.currentRect = tuiPointToClientRect(x, y);
```

Comment:
Why do we need to move it on touchmove? Shouldn't touchmove reset timer as well, meaning we pan rather than longtap?

Reply:
I think it's very difficult not to move your finger at all during a long tap. In android, the browser allows you to move your finger slightly to the side, and the menu will still open. I just simplified this logic a bit.

Reply:
fixed